python_src/src/lamp_py/ingestion/pipeline.py

```diff
@@ -81,6 +85,11 @@ def start() -> None:
         validate_db=True,
     )
 
+    # run metadata rds migrations
+    alembic_upgrade_to_head(db_name="metadata_prod")
+
+    move_metadata()
```

Comment:
temp function to move all the metadata over from rail performance manager. we should only have to do this once.

Reply:
I think we would want to do this in the migration file for the new metadata RDS, which essentially operates as a "temp" function.

Reply:
my instinct was that it would be wrong to have a connection to the RPM database in the migration file as that migration could run when there is no longer a metadata table in that database.